spring-cloud-starter-dataflow-server-local/src/main/resources/dataflow-server.yml

```diff
@@ -37,6 +33,8 @@ spring:
 # that client apps all need a bootstrap.yml (or an environment variable) with the Eureka
 # server address, e.g. in eureka.client.serviceUrl.defaultZone
```

Comment:
wasn't sure, could probably be removed from the starter, although this might be where anyone crating their own server might look